src/azul/chalice.py

```diff
@@ -468,12 +471,34 @@ def catalog(self) -> str:
     def _controller(self, controller_cls: Type[C], **kwargs) -> C:
         return controller_cls(app=self, **kwargs)
 
+    def content_security_policy(self, nonce: Optional[str] = None) -> str:
+        def q(s: str) -> str:
```

Comment:
Is it possible for the string to contain quote characters that need to be escaped?

Consider creating a dedicated, non-local function for this in `src/azul/strings.py`. I'm pretty sure there are other places in the codebase where we use this logic.